### ISSUE TITLE
sometimes proxy container did not start

### ISSUE THREAD

**Issue by hoatle** [teracyhq/dev#425]
nginx proxy is enabled with the chef json config

Steps to reproduce:
- $ docker stop nginx-container

and then `$ vagrant reload --provision`

expected: nginx-proxy should get running

actual: it's not running, with the following similar error:

```
==> default: [2018-03-23T07:50:00+00:00] ERROR: docker_container[nginx-proxy] (teracy-dev::proxy line 84) had an error: Docker::Error::TimeoutError: Container nginx-proxy failed to change to running state after 20 seconds
```

**Comment by hoatle** [comment]
workaround: I must `$ docker rm nginx-proxy` and then `$ vagrant reload --provision` to get it up running as normal.

**Comment by hoatle** [comment]
low priority for now as we're moving forward k8s instead of docker, docker-compose for dev setup.

We'll eventually remove this.

**Comment by hoatle** [comment]
closed, invalid for v0.6.0
